sprint7/project/style.css

```diff
@@ -0,0 +1,49 @@
+body {
+  background-color: rgba(237, 201, 130, 0.85);
+  margin: auto;
+  padding: auto;
+}
+
+.parent {
+  display: flex;
+}
+
+.parent__child1 {
+  width: 20%;
+  padding: 35px;
+  font-family: 'Courier New', Courier, monospace;
+  background-color: rgb(244, 211, 93);
+  text-align: center;
+}
+
+.parent__child2 {
+  width: 20%;
+  padding: 35px;
+  font-family: 'Courier New', Courier, monospace;
+  background-color: rgb(219, 91, 74);
+  text-align: center;
+}
+
+.parent__child3 {
+  width: 20%;
+  padding: 35px;
+  font-family: 'Courier New', Courier, monospace;
+  background-color: rgb(226, 187, 127);
+  text-align: center;
+}
+
+.parent__child4 {
+  width: 20%;
+  padding: 35px;
+  font-family: 'Courier New', Courier, monospace;
+  background-color: rgb(246, 123, 125);
+  text-align: center;
+}
+
+.parent__child5 {
+  width: 20%;
+  padding: 35px;
+  font-family: 'Courier New', Courier, monospace;
+  background-color: rgb(161, 119, 90);
+  text-align: center;
+}
```

Comment:
Poprawki:
1) W tym momencie mamy 5 klas i w każdej z nich powtarzający się kod. To strata zasobów. Proszę powtarzający się kod wsadzić do jednej klasy i tę klasę przypisać każdemu elementowi i dodatkowo dla każdego elementu innego (innego koloru) dodać klasę MODYFIKATOR (BEM: https://www.nafrontendzie.pl/metodyki-css-2-bem). Nazwy klas modyfikatorów niech nie będą parent_child1 tylko nazwa niech bęzie związana z tym za co jest klasa odpowiedzialna
2) Szerokość divów niech będzie obliczana przez przeglądark a nie na sztywno
3) Ustawia pani teraz padding z każrej strony pomimo tego że wszystko jest wycentrowane. Nie potrzeba paddingów z prawej i lewej :)
3) Zmienić piksele na remy :)  https://www.sitepoint.com/understanding-and-using-rem-units-in-css/
4) Elementy zgdonie z projektem mają być na dole strony (FLEX)